content/en/docs/reference/access-authn-authz/authorization.md

```diff
@@ -192,22 +192,41 @@ The following flags can be used:
 You can choose more than one authorization module. Modules are checked in order
 so an earlier module has higher priority to allow or deny a request.
 
-## Privilege escalation via pod creation
+## Privilege escalation via workload creation or edits {#privilege-escalation-via-pod-creation}
 
-Users who have the ability to create pods in a namespace can potentially
-escalate their privileges within that namespace.  They can create pods that
-access their privileges within that namespace. They can create pods that access
-secrets the user cannot themselves read, or that run under a service account
-with different/greater permissions.
+Users who have the ability to create or edit anything which creates pods in a
+namespace can potentially escalate their privileges within that namespace.
+
+
+They can create pods that access other privileges within that namespace. They can
```

Comment:
this paragraph is duplicating the list of escalation paths below. I'd remove it as the escalation paths section is much easier to read